Miki.Discord.Common/Models/IApiClient.cs

```diff
@@ -45,23 +45,23 @@ public interface IApiClient : IGatewayApiClient
 
 		Task<DiscordChannelPacket> GetChannelAsync(ulong channelId);
 
-		Task<List<DiscordChannelPacket>> GetChannelsAsync(ulong guildId);
+		Task<IReadOnlyList<DiscordChannelPacket>> GetChannelsAsync(ulong guildId);
```

Comment:
Regarding these; are we going for eumerables to then move to IAsyncEnumerables? or keep them as lists for now?

Reply:
I've changed them to enumerables.